2021-09-02 20:43:35,633 INFO    MainThread:10549 [wandb_setup.py:_flush():69] setting env: {}
2021-09-02 20:43:35,633 INFO    MainThread:10549 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-02 20:43:35,634 INFO    MainThread:10549 [wandb_init.py:_log_setup():342] Logging user logs to /home/jungwoo/catkin_ws/src/RL_practice/DQN/wandb/run-20210902_204335-1u7htjlh/logs/debug.log
2021-09-02 20:43:35,634 INFO    MainThread:10549 [wandb_init.py:_log_setup():343] Logging internal logs to /home/jungwoo/catkin_ws/src/RL_practice/DQN/wandb/run-20210902_204335-1u7htjlh/logs/debug-internal.log
2021-09-02 20:43:35,634 INFO    MainThread:10549 [wandb_init.py:init():375] calling init triggers
2021-09-02 20:43:35,635 INFO    MainThread:10549 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.001, 'gamma': 0.97}
2021-09-02 20:43:35,635 INFO    MainThread:10549 [wandb_init.py:init():424] starting backend
2021-09-02 20:43:35,635 INFO    MainThread:10549 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-02 20:43:35,663 INFO    MainThread:10549 [backend.py:ensure_launched():135] starting backend process...
2021-09-02 20:43:35,675 INFO    MainThread:10549 [backend.py:ensure_launched():139] started backend process with pid: 10563
2021-09-02 20:43:35,676 INFO    MainThread:10549 [wandb_init.py:init():429] backend started and connected
2021-09-02 20:43:35,678 INFO    MainThread:10549 [wandb_init.py:init():477] updated telemetry
2021-09-02 20:43:35,678 INFO    MainThread:10549 [wandb_init.py:init():500] communicating current version
2021-09-02 20:43:36,172 INFO    MainThread:10549 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-02 20:43:36,172 INFO    MainThread:10549 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-02 20:43:36,547 INFO    MainThread:10549 [wandb_init.py:init():540] starting run threads in backend
2021-09-02 20:43:37,851 INFO    MainThread:10549 [wandb_run.py:_console_start():1584] atexit reg
2021-09-02 20:43:37,852 INFO    MainThread:10549 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-09-02 20:43:37,852 INFO    MainThread:10549 [wandb_run.py:_redirect():1463] Redirecting console.
2021-09-02 20:43:37,853 INFO    MainThread:10549 [wandb_run.py:_redirect():1519] Redirects installed.
2021-09-02 20:43:37,853 INFO    MainThread:10549 [wandb_init.py:init():565] run started, returning control to user process
2021-09-02 20:43:38,939 INFO    MainThread:10549 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 255
2021-09-02 20:43:38,939 INFO    MainThread:10549 [wandb_run.py:_restore():1526] restore
2021-09-02 20:43:41,903 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 808
  total_bytes: 808
}

2021-09-02 20:43:42,254 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 808
  total_bytes: 808
}

2021-09-02 20:43:42,610 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 808
  total_bytes: 22833
}

2021-09-02 20:43:42,711 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 808
  total_bytes: 22833
}

2021-09-02 20:43:42,812 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 808
  total_bytes: 22833
}

2021-09-02 20:43:42,915 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 808
  total_bytes: 22833
}

2021-09-02 20:43:43,019 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 808
  total_bytes: 22833
}

2021-09-02 20:43:43,122 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22833
  total_bytes: 22833
}

2021-09-02 20:43:43,226 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22833
  total_bytes: 22833
}

2021-09-02 20:43:43,330 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22833
  total_bytes: 22833
}

2021-09-02 20:43:43,434 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22833
  total_bytes: 22833
}

2021-09-02 20:43:43,536 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22833
  total_bytes: 22833
}

2021-09-02 20:43:43,638 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22833
  total_bytes: 22833
}

2021-09-02 20:43:44,195 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22833
  total_bytes: 22833
}

2021-09-02 20:43:44,296 INFO    MainThread:10549 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22833
  total_bytes: 22833
}

2021-09-02 20:43:45,382 INFO    MainThread:10549 [wandb_run.py:_show_files():1898] logging synced files
